Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+
+En protégeant les données contre les altérations non autorisées, comme dans le cas où un voleur voudrait corrompre votre appareil SportShield, SportShield garantit l'intégrité des informations critiques.
+
+<br><br>
+
+## Glossaire
+
+Une **carte à communication en champ proche (NFC)** est un type de carte à puce équipée de la technologie NFC. Elle contient une puce NFC intégrée qui lui permet de communiquer sans fil avec des dispositifs compatibles NFC, tels que les smartphones, les terminaux d'identification et de paiement, sur de courtes distances (généralement quelques centimètres).
+
+Un **câble multicouche** est un type de câble électrique composé de plusieurs couches de matériaux différents, chacune ayant une fonction spécifique. Ces câbles sont couramment utilisés dans diverses applications où des caractéristiques de performance spécifiques sont requises.
+
+Une **serrure électromagnétique**, également connue sous le nom de "maglock", est un type de dispositif de verrouillage couramment utilisé dans les systèmes de contrôle d'accès pour sécuriser les portes et les portails. Elle se compose d'un électro-aimant et d'une plaque d'armature. Lorsque l'électro-aimant est alimenté, il génère un champ magnétique qui attire et retient la plaque d'armature, verrouillant ainsi le dispositif en toute sécurité.
+
+L'**indice de protection IP** est un système de classification standard utilisé pour définir le degré de protection d'un boîtier contre l'intrusion d'objets solides tels que la poussière et l'eau. Plus l'indice IP est élevé, plus la protection contre la pénétration de solides et de liquides est importante.
+
+Le **coefficient de résistance à la température (TCR)** est une mesure de l'évolution de la résistance électrique d'un matériau en fonction de la température. Le TCR est généralement exprimé en parties par million par degré Celsius (ppm/°C) ou par Kelvin (ppm/K). Dans le contexte de la résistance au froid, les matériaux à faible TCR sont souhaitables. Inversement, dans le contexte de la résistance à la chaleur, les matériaux ayant des indices TCR plus élevés peuvent être plus appropriés.
+
+<div style="page-break-after: always;"></div>
+
+Ce guide est fourni à titre de référence uniquement. Le produit réel, y compris la couleur, la taille et la disposition de l'écran, peut différer. Les déclarations, informations et recommandations contenues dans ce guide ne constituent pas une garantie, expresse ou implicite.
+<br><br>
+©2015-2024 Coris Innovation. Tous droits réservés. Coris Innovation et le logo Coris Innovation sont des marques commerciales ou des marques déposées de Coris Innovation.
+
+<br><br>
+
+<p align="center">
+    <img src="https://assets-global.website-files.com/656097f111c9c42578f4fa01/65648d5d98f6814022b9474a_6e5ca96fcf317623caec20ca96892327.png" width="300">
```

Comment:
As in the English document, avoid using external images to avoid having dead images afterwards.